test-requirements.txt

```diff
@@ -2,4 +2,4 @@ pytest==3.5.1
 mock==2.0.0
 pytest-xdist==1.20.1
 pytest-cov==2.5.1
```

Comment:
Can we try updating these packages to their most recent version?

Reply:
Fixed.